selfdrive/ui/qt/offroad/networking.cc

```diff
@@ -41,6 +41,7 @@ Networking::Networking(QWidget* parent, bool show_advanced) : QFrame(parent) {
   connect(wifiWidget, &WifiUI::connectToNetwork, this, &Networking::connectToNetwork);
 
   ScrollView *wifiScroller = new ScrollView(wifiWidget, this);
+  wifiScroller->setStyleSheet("background-color: #292929;");
```

Comment:
do we really need to set this everywhere? any docs on how this impacts perf or did you use something like GammaRay?

Reply:
There is no clear documentation. I made these changes through my own tests and by reading Qt source code.